tests/get-criteria-test.js

```diff
@@ -55,12 +55,48 @@ describe('Criteria methods', function() {
         customTargetingKeyId: '597016'
       };
 
+      return expect(dfp.getCriteriaValueId(conditions))
+        .to.eventually.deep.equal('126109156816');
+    });
+
+  });
+
+  describe('getCriteriaValue', function() {
```

Comment:
Could also leave a note here (and any other related tests) to say that this method is deprecated.